.gitignore

```diff
@@ -2,4 +2,6 @@ node_modules
 .nyc_output
 coverage
 .DS_store
-docker-compose.ovverride.yml
\ No newline at end of file
+docker-compose.ovverride.yml
+admin/frontend/assets/styles/
```

Comment:
I think that those files should be in the repo. After introducing gulp without them there will be no styles